projects.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: page
+title: Projects
+parent: Get Involved
+order: 1
+---
+
+Please ask in #welcome:tomesh.net if you would like to learn more or contribute to any one of these projects.
+
+## Nodes/Network
+
+### Raspberry/Orange Pi Install Script + Bi-Weekly Virtual Mesh
+Description: Installation script for Raspbery and Orange Pi devices from a stock firmware.   
+Status: See [Github Issues](https://github.com/tomeshnet/prototype-cjdns-pi/issues)  
+Contact: benhylau  
+[More Info](https://github.com/tomeshnet/prototype-cjdns-pi2)
+
+
+### Mesh Orange
+Description: Ramdisk based os for keeping a lean and mean node on Orange Pis  
+Status: Work in progess https://github.com/tomeshnet/mesh-orange/issues/1  
+Contact: benhylau  
```

Comment:
Could we link to user names in some way? (maybe github?) (indicate talking to people on Chat or where convos happen?)